milestones.tex

```diff
@@ -128,6 +128,19 @@ \subsubsection{Milestones}\label{sec:milestones}
   as verified by a comparison study with the baseline at the beginning of the project.
   }
 
+  \milestone[
+    id=fully-operational,
+    month=36,
+    wps={impact,applications},
+    verif={Services are running and can be accessed and used by users.},
+    ]
+  {Science demonstrators fully operational}
+  {
+  Science demonstrator services are fully operational, enabled by all the developments of the project,
+  and in active use.
+  The most successful demonstrators are prepared for operation beyond the life of the project.
+  }
+
```

Comment:
Reads well. Some of our demonstrators will not be services, such as those using Binder@home or Binder@HPC. Can we include them here?

Reply:
"Science demonstrator applications and services" ?